test/functional/feature_llmq_chainlocks.py

```diff
@@ -243,6 +245,15 @@ def run_test(self):
             assert_equal(tip_1['cbTx']['bestCLSignature'], tip_0['cbTx']['bestCLSignature'])
             assert_equal(tip_1['cbTx']['bestCLHeightDiff'], tip_0['cbTx']['bestCLHeightDiff'] + 1)
 
+        self.log.info(f"Test that bestCLHeightDiff conditions are relaxed before mn_rr")
```

Comment:
nit: no need to add here "f..." because there no any format-string, just a regular string